docs/variants/protectli_ptx01/overview.md

```diff
@@ -0,0 +1,28 @@
+# Overview
+
+The Vault is a small form network appliance built for use as a firewall /
+router, virtualization platform, a daily-driven personal computer, and more.
```

Comment:
```suggestion
router, virtualization platform, a daily-driver personal computer, and more.
```

Reply:
Daily-driven is correct. It is sourced from Protectli page: https://eu.protectli.com/product/vp2410/